### PR TITLE
New version check: error on null/empty

### PR DESCRIPTION
**Pull request by Ahrotahn**
This PR fixes a bug with the new version check action mentioned in #2347 using the fix suggested by @rany2.

### CODE REVIEW

**Review by networkException — APPROVED** [review]

**Comment by rany2** [comment]
@networkException see my comment above, this doesn't work...

**Comment by rany2** [comment]
gah, I didn't submit my comment which is why you didn't see it! it was "pending"

File: .github/workflows/new_version_check.yml [hunk 1]
```diff
@@ -23,9 +23,9 @@ jobs:
       - name: Get the latest Chromium version
         id: latest-version
         run: |
```
[the remark this answers is not in the document]
Comment (Ahrotahn) [reply .github/workflows/new_version_check.yml:25]: From what I can tell from [their documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell) it seems that `-e` is already set by default.

[the remark this answers is not in the document]
Comment (rany2) [reply .github/workflows/new_version_check.yml:25]: That's only for when you pass a single command, if it's multiple lines it doesn't.

**Review by networkException — APPROVED** [review]

**Review by rany2 — APPROVED** [review]

**Review by PF4Public — CHANGES_REQUESTED** [review]
Please don't merge this PR just yet. I'll explain in a moment.

**Comment by PF4Public** [comment]
Sorry everyone for stopping this PR. The issue is that with this workflow we sometimes run into secondary limits for some reason, which results in GitHub spamming failed workflow notifications. I remember this was happening even on 2 hour intervals. I kinda got used to them (albeit it is a bit annoying, since I care for failed workflows in _my_ repository). My concern was that if Ahrotahn merges this PR, this could affect where GitHub sends the notifications. But it turned to be a no issue as Ahrotahn kindly browsed the documentation and found out that only changing cron schedule should divert notifications.

Not an issue with this PR per se, but sorry @rany2 if you took it personally.

Now the problem itself could be solved in the action we are using. There is already an issue filled: https://github.com/dblock/create-a-github-issue/issues/39. Solving it probably requires some experience with Node.js and JavaScript (of which I don't have enough). I'd be grateful if someone could solve it :)

**Comment by rany2** [comment]
>Not an issue with this PR per se, but sorry @rany2 if you took it personally.

No no, I didn't take anything personally. It's just that you said you'll "explain in a moment" but didn't for a couple of hours. So I assumed you did a review like me but forgot to mark it as complete :)

**Pull request merged**